### ISSUE TITLE
Nperf: waiting to close 

### ISSUE THREAD

**Issue by DuredhelFinceleb** [Ysurac/openmptcprouter#447]
## Expected Behavior

After reading that nPerf was better than Speedtest, I gave it a try.
I've tested it with and without OMR
nPerf should chain the DL test, the UL test and the ping test flawlessly

## Actual Behavior
When using OMR, nPerf stops for 2m30s after the DL test with the messeage 'Wating to close'
After that it completes both the UL and ping tests without hiccups.
![image](https://user-images.githubusercontent.com/9292585/56720225-a6799880-6742-11e9-9d28-0ca85358ceaa.png)

## Steps to Reproduce the Problem

  1. ensure OMR is active
  2. go to nPerf website
  3. perform the speedtest

## Specifications

  - OpenMPTCProuter version: openmptcprouter v0.48 r0+9745-b0395cfc56 / LuCI Master (git-19.095.24565-19d15f2) 
  - OpenMPTCProuter VPS version: < OpenMPTCProuter VPS 0.992 > FirstHeberg VPS 200Mb/s
  - OpenMPTCProuter platform: Raspberry Pi 3 Model B Plus Rev 1.3

**Comment by Ysurac** [comment]
I'm not able to reproduce the issue. You have the problem at each try ?

**Comment by DuredhelFinceleb** [comment]
Yes, every single time

**Comment by Lemax9** [comment]
Me too, the same problem
(with the same configuration VPS + Tpi3 b+)
but not with speedtest.telstra.com or fast.com or meter.net

**Comment by DuredhelFinceleb** [comment]
I also have the issue with GOG Galaxay client update process.
It will freeze its download for about 2 to 3 mn at 2.5% then at 31.5% then at 52% and at 87.7% 
And finally it ends up with a download failed process... I tried reinstalling it using a fresh client from the website but at the next client update, it behaved exactly the same way
Weird thing is the issue is not present when Galaxy downloads games. 

I'm quite new to all this... What data can we provided you that might help you figuring out what's happening? And how to get those data?
